components/sms-mgt/rg.wso2.carbon.identity.event.sms.notification.handler/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>identity-event-handler-notification</artifactId>
+        <groupId>org.wso2.carbon.identity.event.handler.notification</groupId>
+        <version>1.7.3</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>rg.wso2.carbon.identity.event.sms.notification.handler</artifactId>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+</project>
```

Comment:
add a new line.

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/commit/6feda46da33ad906dc43f6c786db85b7f0bce845